.github/workflows/issue-label-assign.yml

```diff
@@ -0,0 +1,16 @@
s":["custom-resources","custom resources"],"labels":["@aws-cdk/custom-resources"],"assignees":["eladb"]},{"keywords":["cx-api","cx api"],"labels":["@aws-cdk/cx-api"],"assignees":["eladb"]},{"keywords":["region-info","region info"],"labels":["@aws-cdk/region-info"],"assignees":["eladb"]},{"keywords":["@aws-cdk/aws-acmpca","aws-acmpca","acmpca"],"labels":["@aws-cdk/aws-acmpca"],"assignees":["skinny85"]},{"keywords":["@aws-cdk/aws-appconfig","aws-appconfig","app config","appconfig"],"labels":["@aws-cdk/aws-appconfig"],"assignees":["MrArnoldPalmer"]},{"keywords":["@aws-cdk/aws-cassandra","aws-cassandra","cassandra"],"labels":["@aws-cdk/aws-cassandra"],"assignees":["iliapolo"]},{"keywords":["@aws-cdk/aws-ce","aws-ce"],"labels":["@aws-cdk/aws-ce"],"assignees":["MrArnoldPalmer"]},{"keywords":["@aws-cdk/aws-chatbot","aws-chatbot","chatbot"],"labels":["@aws-cdk/aws-chatbot"],"assignees":["MrArnoldPalmer"]},{"keywords":["@aws-cdk/aws-codeguruprofiler","aws-codeguruprofiler","code guru profiler","codeguruprofiler","codeguru profiler"],"labels":["@aws-cdk/aws-codeguruprofiler"],"assignees":["skinny85"]},{"keywords":["@aws-cdk/aws-detective","aws-detective","detective"],"labels":["@aws-cdk/aws-detective"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-eventschemas","aws-eventschemas","event schemas","eventschemas"],"labels":["@aws-cdk/aws-eventschemas"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-fms","aws-fms","fms"],"labels":["@aws-cdk/aws-fms"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-globalaccelerator","aws-globalaccelerator","global accelerator","globalaccelerator"],"labels":["@aws-cdk/aws-globalaccelerator"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-imagebuilder","aws-imagebuilder","image builder","imagebuilder"],"labels":["@aws-cdk/aws-imagebuilder"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-networkmanager","aws-networkmanager","network manager","networkmanager"],"labels":["@aws-cdk/aws-networkmanager"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-resourcegroups","aws-resourcegroups","resource groups","resourcegroups"],"labels":["@aws-cdk/aws-resourcegroups"],"assignees":["MrArnoldPalmer"]},{"keywords":["@aws-cdk/aws-synthetics","aws-synthetics","synthetics"],"labels":["@aws-cdk/aws-synthetics"],"assignees":["rix0rrr"]},{"keywords":["@aws-cdk/aws-apigatewayv2","aws-apigatewayv2","apigateway v2","apigatewayv2"],"labels":["@aws-cdk/aws-apigatewayv2"],"assignees":["nija-at"]},{"keywords":["@aws-cdk/aws-lambda-nodejs","aws-lambda-nodejs","lambda-nodejs","lambda nodejs"],"labels":["@aws-cdk/aws-lambda-nodejs"],"assignees":["eladb"]},{"keywords":["@aws-cdk/cloudformation-include","cloudformation-include","cloudformation-include","cloudformation include"],"labels":["@aws-cdk/cloudformation-include"],"assignees":["skinny85"]},{"keywords":["@aws-cdk/cloud-assembly-schema","cloud-assembly-schema","cloud-assembly-schema","cloud assembly schema"],"labels":["@aws-cdk/cloud-assembly-schema"],"assignees":["eladb"]}]'
```

Comment:
I think for the sake of readability, this "parameters" snippet should be divided over multiple lines. The [folding style](https://www.baeldung.com/yaml-multi-line#folded-style) yml value division should work.

😸 😷

Reply:
Updated!